### DISCUSSION TITLE
Arch Linux 桌面环境的一些设置 - Muxiner's Blog

### DISCUSSION THREAD

**Discussion by giscus** [Muxiner/muxiner.github.io#25] [博客文章评论]
# Arch Linux 桌面环境的一些设置 - Muxiner's Blog

暂无

https://muxiner.github.io/arch-kde-plasma/

**Comment by Muxiner** [comment]
# 测试
简简单单发个测试试试。

# 代码

```rust
fn main() {
    println!("World, Hello, Rust, ShaBee.");
}
```

# 图片
**简简单单来个图**：

![](https://raw.githubusercontent.com/Muxiner/BlogImages/main/posts/20221101150411-agnoster.png)
